pkg/devserver/service.go

```diff
@@ -315,27 +315,25 @@ func (d *devserver) HandleEvent(ctx context.Context, e *event.Event) (string, er
 	return trackedEvent.GetInternalID().String(), err
 }
 
-type SnapshotValue struct {
-	Type  string      `json:"type"`
-	Value interface{} `json:"value"`
-}
-
 func (d *devserver) exportRedisSnapshot(ctx context.Context) (err error) {
 	d.snapshotLock.Lock()
 	defer d.snapshotLock.Unlock()
 
-	snapshot := make(map[string]SnapshotValue)
+	var (
+		snapshotID cqrs.SnapshotID
+		snapshot   = make(map[string]cqrs.SnapshotValue)
+		l          = logger.From(ctx).With().Str("caller", "devserver").Logger()
+	)
 
-	l := logger.From(ctx).With().Str("caller", "devserver").Logger()
 	l.Info().Msg("exporting Redis snapshot")
 	defer func() {
 		if err != nil {
 			l.Error().Err(err).Msg("error exporting Redis snapshot")
-		} else {
-			jsonData, _ := json.Marshal(snapshot)
-			humanSize := fmt.Sprintf("%.2fKB", float64(len(jsonData))/1024)
-			l.Info().Str("size", humanSize).Msg("exported Redis snapshot")
 		}
+
+		jsonData, _ := json.Marshal(snapshot)
+		humanSize := fmt.Sprintf("%.2fKB", float64(len(jsonData))/1024)
+		l.Info().Str("size", humanSize).Str("snapshot_id", snapshotID.String()).Msg("exported Redis snapshot")
```

Comment:
**question**: I wonder if we should call this "queue snapshot" in logs or something like that ("job queue", "queue and state", "runs snapshot"). It doesn't matter for the sake of this PR, just thinking about decoupling the data and backing service type (even if it's in-memory) from the user-facing logs.